bases/sfmc-app/@{{ cookiecutter.project_slug }}/pm2.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
    apps: [
       {
-         name: "@{{ service_name }}",
+         name: "@{{ cookiecutter.service_name }} }}",
```

Comment:
are the double closing bracket typos or needed?
```suggestion
         name: "@{{ cookiecutter.service_name }}",
```